src/linearity.jl

```diff
@@ -151,17 +151,17 @@ function detect_new_linearities(rep::Representation, solver; verbose=0, kws...)
     nonlins = _nonlinearity(rep)
     isempty(nonlins) && return Int[]
     model, T = layered_optimizer(solver)
-    verbose >= 2 && (_model = JuMP.direct_model(model))
+    # verbose >= 2 && (_model = JuMP.direct_model(model))
     is_lin = falses(length(nonlins))
     active = trues(length(nonlins))
     # We pass `true` as we break homogeneity of the problem with `sum(λ) == 1`.
     hull = _zero_hull(rep, T)
     λ, cλ, sum_con = _hull(model, T, hull, rep, eachindex(nonlins), true)
-    if verbose >= 2
-        for (i, λ) in enumerate(λ)
-            MOI.set(model, MOI.VariableName(), λ, "λ[$i]")
-        end
-    end
+    # if verbose >= 2
+    #     for (i, λ) in enumerate(λ)
+    #         MOI.set(model, MOI.VariableName(), λ, "λ[$i]")
+    #     end
+    # end
```

Comment:
Why was this commented out ?

Reply:
Yes, when I wrote that, MathOptInterface didn't have printing so I had to create a JuMP model. Now, MOI has its own printing: https://jump.dev/MathOptInterface.jl/dev/submodules/Utilities/overview/#Printing so we can just remove this `direct_model`